src/main/java/gov/loc/repository/bagit/conformance/ManifestChecker.java

```diff
@@ -20,134 +20,134 @@
 import gov.loc.repository.bagit.util.PathUtils;
 
 /**
- * Part of the BagIt conformance suite. 
- * This checker checks for various problems related to the manifests in a bag.
```

Comment:
as far as I can tell, this file only contains formatting changes, is that correct?